.circleci/config.yml

```diff
@@ -75,17 +68,66 @@ jobs:
             terraform workspace select default
             terraform workspace delete ${WORKSPACE}
 
-parameters:
-  terraform_version:
-    default: "1.3.6"
-    type: string
+  test-upgrade:
+    docker:
+      - image: cimg/aws:2022.09
```

Comment:
@miguelhar Have we (or can we) try testing this with the [latest](https://circleci.com/developer/images/image/cimg/aws) (`2022.11.1`)? If it causes issues, I'm okay with staying on `2022.09`